shell/platform/windows/BUILD.gn

```diff
@@ -55,26 +61,41 @@ source_set("flutter_windows_source") {
     "string_conversion.cc",
     "string_conversion.h",
     "system_utils.h",
-    "system_utils_win32.cc",
+    "task_runner.h",
     "text_input_plugin.cc",
     "text_input_plugin.h",
-    "win32_dpi_utils.cc",
-    "win32_dpi_utils.h",
-    "win32_flutter_window.cc",
-    "win32_flutter_window.h",
-    "win32_platform_handler.cc",
-    "win32_platform_handler.h",
-    "win32_task_runner.cc",
-    "win32_task_runner.h",
-    "win32_window.cc",
-    "win32_window.h",
-    "win32_window_proc_delegate_manager.cc",
-    "win32_window_proc_delegate_manager.h",
     "window_binding_handler.h",
     "window_binding_handler_delegate.h",
     "window_state.h",
   ]
 
+  # Target-specific sources.
+  if (target_os == "winuwp") {
+    sources += [
+      "flutter_windows_winuwp.cc",
+      "system_utils_winuwp.cc",
+      "winrt_task_runner.cc",
+      "winrt_task_runner.h",
```

Comment:
Can we be consistent about winrt vs winuwp, or is the distinction meaningful? I.e., is there a scenario where this file would be used for non-UWP? (If I look up the CoreWindow code there I get docs in the UWP section of docs.microsoft.com.)

Reply:
Agree that things aren't 100% consistent, i've tried to use winrt to refer to classes that exclusively call winrt API's and not win32.  There will be cases in the future where WinRT API's are called from the desktop version of some classes hence maybe it would be clearer to use UWP

Reply:
I'll standardize on UWP; I think that since we generally call them the Win32 embedding and the UWP embedding, we should use those labels for files that are only for one or the other (even if we potentially end up with things like "win32" files that use WinRT--but hopefully over time migration to WinRT would come with unforking classes to use a shared Reunion API?)

Reply:
Sounds good.  Yes, that should be the case